src/main/resources/META-INF/plugin.xml

```diff
@@ -8,8 +8,6 @@
     <extensions defaultExtensionNs="com.intellij">
         <applicationService serviceImplementation="com.drinchev.projectlabel.preferences.ApplicationPreferences"/>
         <projectService serviceImplementation="com.drinchev.projectlabel.preferences.ProjectPreferences"/>
-        <projectService serviceImplementation="com.drinchev.projectlabel.ProjectLabel" />
-        <projectService serviceImplementation="com.drinchev.projectlabel.resources.ui.ProjectLabelBackgroundImage" />
```

Comment:
are these obsolete due to the annotations?

Reply:
No clue, why exactly, but once I moved the class of the service to `final` and IntelliJ suggested that since this is not a complex component I can leave these out.